Jenkinsfile

```diff
@@ -1201,6 +1202,7 @@ pipeline {
                     }
                     steps {
                         testRpm inst_repos: el7_daos_repos(),
+                                inst_rpms: test_rpm_rpms,
```

Comment:
This is not an avocado driven test.

Reply:
It is not, but the `config_gile_gen.py` script it uses to generate the configuration files for the `daos_agent` and `daos_server` commands it uses to test the DAOS RPMs (as part of this PR) imports (but does not really need to use for the purpose of testing the DAOS RPMs) the avocado.utils python library.

We can avoid edits to this file and the import problems (related to the current `command_utils_base.py` updates), by instead moving the code in `command_utils_base.py` that imports `general_utils.py` to the `command_utils.py` file (not needed/imported by `config_fiel_gen.py`).

Reply:
> It is not, but the `config_gile_gen.py` script it uses to generate the configuration files for the `daos_agent` and `daos_server` commands it uses to test the DAOS RPMs (as part of this PR) imports (but does not really need to use for the purpose of testing the DAOS RPMs) the avocado.utils python library.

I see.  It sounds like an avocado-independent version of this tool needs to be written perhaps then, yes?
 
> We can avoid edits to this file and the import problems (related to the current `command_utils_base.py` updates), by instead moving the code in `command_utils_base.py` that imports `general_utils.py` to the `command_utils.py` file (not needed/imported by `config_fiel_gen.py`).

I guess this is accomplishing the above suggestion then?

Reply:
> I guess this is accomplishing the above suggestion then?
yes
 
> In any case, the avocado RPMs being added to `inst_rpms` here are already installed on the test node, so there is no need to specify them here also.
Although this is now inconsequential with the removal of the `import general_utils.py` in `command_utils_base.py`, doesn't the python `ImportError` for `pathlib` indicate otherwise?  Presumably there is nothing special needed in terms of PYTHONPATH setup for `pathlib` - we just install the RPM and it works.

Reply:
I don't know why `pathlib` is not being found, but it is installed even before this change as we can see in the most recent run:

```
[2020-07-18T21:54:35.394Z] + yum -y install CUnit python36-PyYAML python36-nose python36-pip valgrind python36-paramiko python2-avocado python2-avocado-plugins-output-html python2-avocado-plugins-varianter-yaml-to-mux libcmocka python-pathlib python2-numpy git python2-clustershell golang-bin ipmctl ndctl
[2020-07-18T21:54:35.394Z] Loaded plugins: fastestmirror
[2020-07-18T21:54:35.394Z] Loading mirror speeds from cached hostfile
[2020-07-18T21:54:35.394Z]  * base: mirrors.sonic.net
[2020-07-18T21:54:35.394Z]  * epel: mirror.chpc.utah.edu
[2020-07-18T21:54:35.394Z]  * extras: mirrors.gigenet.com
[2020-07-18T21:54:35.394Z]  * updates: mirror.chpc.utah.edu
[2020-07-18T21:54:35.394Z] Package CUnit-2.1.3-8.el7.x86_64 already installed and latest version
[2020-07-18T21:54:35.394Z] Package python36-PyYAML-3.13-1.el7.x86_64 already installed and latest version
[2020-07-18T21:54:35.394Z] Package python36-nose-1.3.7-4.el7.noarch already installed and latest version
[2020-07-18T21:54:35.394Z] Package python3-pip-9.0.3-7.el7_7.noarch already installed and latest version
[2020-07-18T21:54:35.394Z] Package 1:valgrind-3.15.0-11.el7.x86_64 already installed and latest version
[2020-07-18T21:54:35.394Z] Package python36-paramiko-2.1.1-0.10.el7.noarch already installed and latest version
[2020-07-18T21:54:35.394Z] Package python2-avocado-52.1-6.el7.noarch already installed and latest version
[2020-07-18T21:54:35.394Z] Package python2-avocado-plugins-output-html-52.1-6.el7.noarch already installed and latest version
[2020-07-18T21:54:35.394Z] Package python2-avocado-plugins-varianter-yaml-to-mux-52.1-6.el7.noarch already installed and latest version
[2020-07-18T21:54:35.394Z] Package libcmocka-1.1.5-1.el7.x86_64 already installed and latest version
[2020-07-18T21:54:35.394Z] Package python-pathlib-1.0.1-1.el7.noarch already installed and latest version
[2020-07-18T21:54:35.394Z] Package 1:python2-numpy-1.7.1-0.el7.noarch already installed and latest version
[2020-07-18T21:54:35.394Z] Package git-1.8.3.1-23.el7_8.x86_64 already installed and latest version
[2020-07-18T21:54:35.394Z] Package python2-clustershell-1.8.3-1.el7.noarch already installed and latest version
[2020-07-18T21:54:35.394Z] Package golang-bin-1.13.11-1.el7.x86_64 already installed and latest version
[2020-07-18T21:54:35.394Z] Package ipmctl-02.00.00.3764-1.el7.x86_64 already installed and latest version
[2020-07-18T21:54:35.394Z] Package ndctl-65-5.el7.x86_64 already installed and latest version
[2020-07-18T21:54:35.394Z] Nothing to do
```